files/default/reaml.groovy

```diff
@@ -0,0 +1,12 @@
+import groovy.json.JsonSlurper
+import org.sonatype.nexus.security.realm.RealmManager
+
+def params = new JsonSlurper().parseText(args)
+
+def realmManager = container.lookup(RealmManager.class.getName())
+
+if (params.action == 'enable') {
```

Comment:
Instead of "action" parameter pass "enable" as Boolean and use it in your call `realmManager.enableRealm(params.name, params.enable)`